UIKitDZ/UIKitDZ/PizzaSelectionViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  PizzaSelectionViewController.swift
+//  UIKitDZ
+//
+//  Created by Дмитрий Супрун on 29.09.22.
+//
+
+import UIKit
+
+// Choosing a pizza from the list
+final class PizzaSelectionViewController: UIViewController {
+    
+    // MARK: - Public Properties
+    
+    var pizza: [Pizza] = [Pizza(name: "Margarita", image: "Margarita"),
```

Comment:
```suggestion
    var pizzas: [Pizza] = [Pizza(name: "Margarita", image: "Margarita"),
```
всегда в множественном массив

Reply:
Мне переводчик дал pizza во множественном числе ))